yellowbrick/classifier/classification_report.py

```diff
@@ -99,21 +114,26 @@ def score(self, X, y=None, **kwargs):
 
         y : ndarray or Series of length n
             An array or series of target or class values
-        """
+        """ 
         y_pred = self.predict(X)
 
         scores = precision_recall_fscore_support(y, y_pred)
 
         # Calculate the percentage for the support metric
-        self.support_score = scores[-1]
-        support_percent = self.support_score / (sum(self.support_score))
+        # and store the percent in place of raw support counts
+        self._support_score = scores[-1]
+
+        scores = list(scores)
+        scores[-1] = scores[-1] / scores[-1].sum()
```

Comment:
Ok, so you've chosen to store the original support scores and use `scores_` to only store intensity values. I think that's perfectly reasonable. In this case, however, I would change the variable name to `self.support_score_` -- `self._support_score` indicates something we'd rather not let the user take a look at (it's internal, protected), whereas `self.support_score_` indicates a learned parameter that might be important to the user, particularly if they specify percentage mode.